README.md

```diff
@@ -301,20 +301,20 @@ of any of the guilds available to the bot.
 
 ### Entity Updates
 
-When an entity is updated through its manager they will send a request to the Discord API which will update the state
-of the entity. The success of this request **does not** imply the entity has been updated yet. All entities are updated
+When an entity is updated through its manager, they will send a request to the Discord API which will update the state
+of the entity. The success of this request **does not** imply the entity has finished updating. All entities are updated
```

Comment:
You're changing the meaning of the sentence here, and the new meaning is actually less explicit. The update might not even have started, hence the wording.